core/controllers/suggestion.py

```diff
@@ -139,6 +187,58 @@ def put(self, target_id, suggestion_id):
         self.render_json(self.values)
 
 
+class ReviewableSuggestionsHandler(base.BaseHandler):
+    """Provides all suggestions which can be reviewed by the user for a given
+    suggestion type.
+    """
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
```

Comment:
I don't think this is the right permissions for this handler?

Reply:
@nithusha21 As of now, only the admin has the rights to review the suggestions through the community dashboard page, we will add other users for review once we have our scoring system in place.